### PR TITLE
refactor: integrate with sidecar

### PR DESCRIPTION
**Pull request by kyranjamie**
Takes Thomas' work and hooks it up to the sidecar and the types it outputs.

### CODE REVIEW

File: src/components/transaction-details.tsx [hunk 1]
```diff
@@ -63,48 +39,55 @@ const transformDataToRowData = (d: Transaction) => {
...
+      copy: d.sender_address,
     },
     {
       label: {
         children: 'Called by',
       },
-      children: d.recipient,
-      copy: d.recipient,
+      children: d.sender_address,
+      copy: d.sender_address,
```
Comment (aulneau) [inline src/components/transaction-details.tsx:57]: Should these values be the same?

Comment (kyranjamie) [reply src/components/transaction-details.tsx:57]: They were before?

Comment (kyranjamie) [reply src/components/transaction-details.tsx:57]: ah see what you mean now

File: .eslintrc.js [hunk 2]
```diff
@@ -1,73 +1,18 @@
...
-  extends: [
-    'plugin:@typescript-eslint/recommended',
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
-    'plugin:react/recommended',
-    'prettier/@typescript-eslint',
-    'plugin:jest/recommended',
-    'plugin:prettier/recommended'
-    // 'plugin:jsx-a11y/recommended'
-  ],
+  extends: ['@blockstack/eslint-config'],
```
Comment (aulneau) [inline .eslintrc.js:2]: love this

**Pull request merged**